apidoc/Titanium/UI/WebView.yml

```diff
@@ -693,6 +693,12 @@ properties:
     type: Dictionary
     platforms: [android,iphone,ipad]
     since: "6.1.0"
+  - name: mediaPlaybackRequiresUserGesture
+    summary: Sets whether the WebView requires a user gesture to play media.
+    type: Boolean
+    platforms: [android, iphone, ipad]
+    since: "6.2.0"
```

Comment:
Use "7.0.0" instead. 6.2.0 is already shipped, 6.3.0 is already in RC next week + iOS 11 focussed and the next one after that is 7.0.0.